modules/n1ql/pages/n1ql-language-reference/flex-indexes.adoc

```diff
@@ -174,7 +181,7 @@ The full-text index _must_ use one of the following type mappings:
 * A single custom type mapping
 * Multiple custom type mappings
 
-The full-text index may _not_ use the default type mapping along with one more more custom type mappings.
+The full-text index may _not_ use the default type mapping along with one or more custom type mappings.
```

Comment:
Some where  you should mention Only keyword analyzer indexes are considered in Flex Index.

Reply:
Fully explained on [Flex Index](The full-text index must use the keyword analyzer) page:

> The full-text index must use the keyword analyzer